### PR TITLE
CT-2148 closed cases turn up in my open cases tab

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

**Review by mseedat-moj — APPROVED** [review]

**Review by mseedat-moj — DISMISSED** [review]
Request from Stephen to un-approve

File: app/services/case_finder_service.rb [hunk 1]
```diff
@@ -120,19 +122,20 @@ def in_states(states)
     scope.in_states(states.split(','))
   end
 
+  CASE_TYPES_ONLY_VISIBLE_WITH_FURTHER_CLEARANCE = [
```
Comment (beccapearce) [inline app/services/case_finder_service.rb]: Not sure about what this is doing, could we put a comment in?

File: spec/controllers/cases_controller_spec.rb [hunk 2]
```diff
@@ -446,6 +443,26 @@ def sar_closure_params(sar)
...
+      let!(:in_time_case) { create(:accepted_case) }
+      let!(:late_case) { Timecop.freeze(50.days.ago) { create(:accepted_case) } }
+      let!(:closed_case) { create(:closed_case) }
+
+      before do
+        sign_in responder
+      end
+
+      it 'gets my in time cases' do
+        get :my_open_cases, params: { tab: 'in_time' }
```
Comment (beccapearce) [inline spec/controllers/cases_controller_spec.rb]: do we have a similar test for incoming cases? (I'm assuming that's what page was reported in the ticket)

**Review by beccapearce — CHANGES_REQUESTED** [review]
Just a comment and test request

File: app/services/case_finder_service.rb (hunk 1 above)
Comment (starswan) [reply app/services/case_finder_service.rb]: I just moved it from a dynamic attribute to a static one whilst in the code. I don't really know any more

File: spec/controllers/cases_controller_spec.rb [hunk 3]
```diff
@@ -443,36 +444,34 @@ def sar_closure_params(sar)
     end
 
     context 'without stubs' do
-      let!(:in_time_case) { create(:accepted_case) }
-      let!(:late_case) { Timecop.freeze(50.days.ago) { create(:accepted_case) } }
+      # let!(:in_time_case) { create(:accepted_case) }
```
Comment (beccapearce) [inline spec/controllers/cases_controller_spec.rb]: Do we still need this code?

**Review by beccapearce — CHANGES_REQUESTED** [review]
A few commented bits of code left in but otherwise looks good

File: spec/controllers/cases_controller_spec.rb (hunk 3 above)
Comment (starswan) [reply spec/controllers/cases_controller_spec.rb]: no clearly not :-)

**Review by beccapearce — APPROVED** [review]

**Pull request merged**